packages/gallery-indicator/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+const LGID_ROOT = 'lg-gallery_indicator';
+
+export const LGIDS = {
```

Comment:
Add a ticket to make these consistent across all components

Reply:
What's currently inconsistent?